float64 pan_wv
float64 tilt_wv
float64 zoom